<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Resultado</string>
   </property>
  </widget>
  <widget class="QLabel" name="txt_resultado">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
